migrations/2023-07-19-081050_add_zero_decimal_currencies/up.sql

```diff
@@ -1,14 +1,75 @@
 -- Your SQL goes here
+ALTER TYPE "Currency" ADD VALUE IF NOT EXISTS 'AFN' AFTER 'AED';
```

Comment:
Here you cannot edit the old file. Instead you have to create new one.
Use this command. The files will be automatically created - `diesel migration generate <migration_file_name>`.
Two files will be created - `up.sql` should have all these queries
`down.sql` should have `SELECT 1;`